docs/configure.rst

```diff
@@ -60,29 +84,33 @@ the following method::
 More information can be found in the `paste httpserver documentation
 <http://pythonpaste.org/modules/httpserver.html>`_.
```

Comment:
This link doesn't work anymore. Should be replaced by this https://paste.readthedocs.io/en/latest/index.html

Reply:
or should be https://pythonpaste.readthedocs.io/en/latest/. I am a bit puzzled by the two websites.

Reply:
pythonpaste.readthedocs.org seems to be the updated one.

Reply:
That said, the text refers to the module documentation, which isn't being generated on RTD, so I've changed the link to the source.